Driverless AI/image-processing-in-driverless-ai/image-processing-in-driverless-ai.md

```diff
@@ -0,0 +1,595 @@
+
+In the next section, let's explore the pre-rebuilt experiment from task one, and let's see the impact fine-tuning has on the model's performance.
+
+## Task 4: Embeddings Transformer (Image Vectorizer) with Fine-tuning
+
+The experiment from task one has been rerun already (with fine-tuning) because it takes longer than two hours (once again, the Aquarium test drive only runs for two hours). The experiment has been named `Embeddings-Transformer-With-Fine-Tuning`.
+
+To showcase how fine-tuning was enabled for the first approach to image processing in DAI, observe the steps you can take to rerun the experiment with fine-tuning: 
+
+In the **Experiments** section:
+
+1. Click the **three vertical dots** (located on the right side of the experiment) of the following experiment: `Embeddings-Transformer-Without-Fine-Tuning`
+
+![exp2-new-experiment](assets/exp2-new-experiment.png)
+
+2. Click the following option: **NEW EXPERIMENT WITH SAME SETTINGS**
+
+3. Rename the experiment to `Embeddings-Transformer-With-Fine-Tuning`
+
+4. Under the **IMAGE** tab located in the **EXPERT SETTINGS** click the **DISABLED** button under the following setting: **Enable Fine-tuning of pre-trained models used for image Transformer** 
+
+    - This setting will change from **DISABLED** to **ENABLED** 
+
+![enabled-fine-tuning](assets/enabled-fine-tuning.png)
+
+5. **LAUNCH EXPERIMENT**
+
+![exp2-launch-experiment](assets/exp2-launch-experiment.png) 
+
+When fine-tuning is enable, Driverless AI provides a list of possible image augmentations to apply while fine-tuning the **ImageNet pre-trained models** used for the **Image Transformer**. By default, **HorizontalFlip** is enabled, and for purposes of this tutorial, it was not changed.. This default setting can be found and change in the **IMAGE** tab inside the **EXPERT SETTINGS**. Please refer to the Driverless AI documentation right [here](https://docs.h2o.ai/driverless-ai/latest-stable/docs/userguide/expert_settings/image_settings.html#list-of-augmentations-for-fine-tuning-used-for-the-image-transformer) for a full list of all other augmentations. 
+
+![list-of-augmentations-for-fine-tuning-used-for-image-transformer](assets/list-of-augmentations-for-fine-tuning-used-for-image-transformer.png)
+
+**NOTE**: Augmentations for fine-tuning used for the Image Transformer is only available when Fine-tuning is **enabled**.  
+
+As well, when fine-tuning is enabled, you can specify the number of epochs for fine-tuning **ImageNet pre-trained** models used for the **Image Transformer**. This value defaults to **2**. This default setting can be found and change in the **IMAGE** tab inside the **EXPERT SETTINGS**.
+
+Now that you know how to rerun the experiment with fine-tuning let's explore the new experiment (Embeddings-Transformer-With-Fine-Tuning). 
+
+![finish-experiment-car_deals_with_fine_tuning](assets/finish-experiment-car_deals_with_fine_tuning.png)
```

Comment:
It seems that this experiment hasn't used Image features at all in the final model. As it has only 50 features and there is no ImageVectorizer features in the variable importance.

I would try to re-run it with different settings/architecture/target variable in order to see image features here.